k8s/upgrade-job/src/upgrade/path.rs

```diff
@@ -72,3 +65,27 @@ pub(crate) fn version_from_release_chart(chart_name: String) -> Result<Version>
         version_string: version.to_string(),
     })
 }
+
+#[derive(Debug, Serialize, Deserialize)]
+struct UnspportedVesrsions {
+    unsupported_versions: Vec<String>,
+}
+
+pub(crate) fn get_invalid_upgrade_path(path: PathBuf) -> Result<HashSet<Version>> {
```

Comment:
```suggestion
pub(crate) fn invalid_upgrade_path(path: PathBuf) -> Result<HashSet<Version>> {
```

Missing doc comments